### DISCUSSION TITLE
Cabricality 0.3

### DISCUSSION THREAD

**Discussion by KrLite** [DM-Earth/Cabricality#151] [Show and tell]
*Aegroto dum anima est!*

Welcome to **Cabricality 0.3.** Here, everything can be further than what is out of your reach, yet everything is within two steps of your own. It depends **on you!**

Napping beyond dense fresh flower wonderlands, racing to catch up an intensely scheduled underground mining train, floating, walking outside the breathtaking earth atmosphere? **Everything, everywhere, all at Cabricality!**

*Live Long and Prosper, Adventurer!*

<hr /><em>This discussion was created from the release <a href='https://github.com/DM-Earth/Cabricality/releases/tag/0.3'>Cabricality 0.3</a>.</em>

**Comment by KrLite** [comment]
Tasty!

**Comment by Gebardensprache** [comment]
actually working start when

**Reply by KrLite** [reply]
> actually working start when

July 2025

**Reply by Gebardensprache** [reply]
> > actually working start when
> 
> July 2025

its too far and long wait

**Reply by KrLite** [reply]
> > > actually working start when
> 
> > 
> 
> > July 2025
> 
> 
> 
> its too far and long wait

I have to live and study.

I'm not god.
